wesnoth-ei.po

```diff
@@ -195,10 +195,11 @@ msgid ""
 "eye and hand firmly on the settled country to either side of the lower "
 "Weldyn. He sent two of the most promising young officers to them."
 msgstr ""
-"Při objevení nového nepřítele Wesnothu na východě se Konrád II. rozhodl "
-"opravit tyto říční pevnosti a obnovit jejich posádku, aby tak mohla Koruna "
-"lépe sledovat osídlené kraje na východ od Weldynu a v případě potřeby je "
-"podržet silnou rukou. Vyslal na ně své nejslibnější mladé důstojníky."
+"Poté, co začaly na Wesnoth od východu dotírat neznámé nepřátelské síly, se "
+"král Konrád II. rozhodl tyto říční pevnosti opravit a obnovit jejich "
+"posádku. Chtěl tak dosáhnout, aby tak mohla Koruna lépe sledovat osídlené "
+"kraje na východ od Weldynu a v případě potřeby je podržet silnou rukou. "
```

Comment:
v originále je něco jako na obou stranách dolního Weldynu (ne na východ)

Reply:
Správně. Měl jsem před očima vlčí mlhu a četl jsem "east side of the river Weldyn".